### PR TITLE
Document dependencies

### PR DESCRIPTION
**Pull request by adamjstewart**
Adding some missing dependencies.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def run_tests(self):
     ext_package='cmocean',
     scripts = [],
     keywords = ['colormaps', 'oceanography', 'plotting', 'visualization'],
+    setup_requires=['setuptools'],
+    install_requires=['matplotlib', 'numpy'],
```
Comment (tacaswell) [inline setup.py:48]: Is it worth doing the "read the requirements.txt file" trick here (to make sure these never get out of sync)?

Comment (adamjstewart) [reply setup.py:48]: Depends on how often these dependencies will change. Let me know if you want me to do that.

Comment (adamjstewart) [reply setup.py:48]: @kthyng do you have any opinions on this?

Comment (kthyng) [reply setup.py:48]: Ack, sorry that I neglected this. I don't care too much but I do not expect the dependencies to change much if ever!

**Review by kthyng — APPROVED** [review]
This looks fine to me!

**Comment by adamjstewart** [comment]
It looks like the Travis tests are hung. Opening and closing to restart the tests...

**Comment by dopplershift** [comment]
Travis-CI has recently [started limiting resources](https://blog.travis-ci.com/oss-announcement) for open source projects. Matplotlib itself migrated to GitHub Actions, and this repo probably needs to as well.

**Comment by adamjstewart** [comment]
I would be surprised if this repo exceeded the resource limits, it's not particularly active

**Comment by dopplershift** [comment]
Well, they're shared across the org...but I guess with matplotlib itself moving, there are now plenty of resources to go around?

**Comment by adamjstewart** [comment]
Well, Travis finally ran, but it's still erroring out. What do we want to do here?

**Comment by adamjstewart** [comment]
Can we get this merged?

**Comment by kthyng** [comment]
I am not clear on protocol. Can I merge this without messing something else up? If so I will do that.

**Comment by dopplershift** [comment]
@kthyng This repo is independent of the main matplotlib repo, so as the cmocean maintainer, merge whatever you're happy with.

**Comment by kthyng** [comment]
@dopplershift Ah, sorry for the delay then. I figured someone else was deciding.

**Comment by kthyng** [comment]
Thanks @adamjstewart!!

**Pull request merged**